tests/test_acquisition.py

```diff
@@ -0,0 +1,279 @@
+import pytest
+import numpy as np
+from bayes_opt import acquisition
+from sklearn.gaussian_process import GaussianProcessRegressor
+from bayes_opt.target_space import TargetSpace
+from bayes_opt.constraint import ConstraintModel
+from scipy.spatial.distance import pdist
+from bayes_opt import acquisition 
+# TODO: Add tests that checks that the acq_max actually returns the maximum
```

Comment:
should we open a new issue for this rather than leave the ToDo in source?

Reply:
I went ahead and added these tests + deleted the line. They're basic tests (optimizing a 2d quadratic function), but I guess that's all that's needed.